views/js/pciCreator/ims/likertScaleInteraction/imsPciCreator.json

```diff
@@ -0,0 +1,42 @@
+{
+    "model": "IMSPCI",
+    "typeIdentifier": "likertScaleInteraction",
+    "label": "Likert Scale",
+    "description": "Implementation of a Likert scale",
+    "version": "1.0.0",
+    "author": "OAT SA",
+    "email": "contact@taotesting.com",
+    "tags": [
+      "mcq",
+      "likert"
+    ],
+    "icon": "./icon.svg",
+    "short": "Likert Scale",
+    "response": {
+        "baseType": "integer",
+        "cardinality": "single"
+    },
+    "runtime": {
+        "hook": "./runtime/js/likertScaleInteraction.min.js",
+        "src": [ "./runtime/js/likertScaleInteraction.js"],
```

Comment:
Ok. (Even when we are bundling the sources and defining as the `hook` the .min.js file? I don't see how the `src` listings would be used, but I will add them anyway.)

Reply:
It is used by the package manager. Check the content of the portableElementStorage and what is exported with a test/item package. It will help you understand